Makefile

```diff
@@ -30,6 +30,13 @@ docs: .paasta/bin/activate
 test: .paasta/bin/activate
 	.paasta/bin/tox -i $(PIP_INDEX_URL)
 
+dev-api: .paasta/bin/activate
+	@if [ ! -d .tox/py36-linux ] ; \
```

Comment:
I think this is "fine", but isn't really in the spirit of `make`.

Instead, use make's ability to understand dependencies and conditionally run them, just like `.paasta/bin/activate`:

```
.tox/py36-linux:
    .paasta/bin/tox...

dev-api:  .tox/py36-linux
   .tox/py36....
```